src/tcutility/analysis/vdd/manager.py

```diff
@@ -0,0 +1,220 @@
+﻿from __future__ import annotations
+import pandas as pd
+import numpy as np
+from typing import Dict, List, Optional, Sequence, Union
+import attrs
+import pathlib as pl
+from itertools import zip_longest
+import copy
+import functools
+from tcutility.analysis.vdd import charge
+from tcutility.results import result
+import matplotlib.pyplot as plt
+
+
+AVAILABLE_UNITS_RATIO = {"me": 1000, "e": 1}
```

Comment:
Consider moving this to tcutility.constants

Reply:
Done